test/reuse/util/browser/forward.spec.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const browser = require("../../../../lib/reuse/modules/util/browser");
+const { handleCookiesConsent } = require("../../../helper/utils");
+
+describe("Expect browser url to be equal with 'startUrl' after browser forward", function () {
```

Comment:
can you change the description top match the functions name? Maybe also adjust the same for "back" since it was wrong there as well.
```suggestion
describe("browser - forward", function () {
```

Reply:
I will change the description as per our standard

Reply:
done